frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetToRemoteAction.java

```diff
@@ -160,8 +160,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
               remoteTrackingBranch.getName(),
               currentCommitSha));
 
-      dialogBuilder.yesText(getString("action.GitMachete.BaseResetToRemoteAction.info-dialog.ok-text"))
```

Comment:
Relying on defaults here? 🤔

Reply:
![image](https://user-images.githubusercontent.com/19799111/195371862-11f22036-5ccc-47ef-a7dc-9fde9c2a179b.png)

Reply:
by a simple modification of the dialog message we can drop some code and rely on defaults